packages/field-base/src/label-mixin.js

```diff
@@ -16,7 +16,8 @@ const LabelMixinImplementation = (superclass) =>
          */
         label: {
           type: String,
-          observer: '_labelChanged'
+          observer: '_labelChanged',
+          reflectToAttribute: true
```

Comment:
Why do we need this? The `has-label` attribute is enough.

Reply:
Oh, good catch! Removed.